test/viper/unsupported.mo

```diff
@@ -2,6 +2,4 @@
 
 actor {
 
-  let x = ""; // strings aren't supported
-
 }
```

Comment:
I think we should change it to something as well unsupported.
I think the idea of this smoke test is a check of error reporting

Reply:
Not sure that I should do this as a part of this ticket... However, we can add some things from other milestone tickets that aren't implemented yet.

Reply:
replace it with `let x = '' // char aren't supported`

Reply:
Added

Reply:
thx